src/main/resources/application-custom.yaml

```diff
@@ -129,6 +129,11 @@ hapi:
 #    bundle_batch_pool_size: 10
 #    bundle_batch_pool_max_size: 50
 
+    oauth:
```

Comment:
To be backwards compatible, this configuration just reads from the environment. We'll change it when we start using custom application.yaml files for different servers.